{
    "status": "failed", 
    "name": "test_sequence_of_outside_steps[True-False]", 
    "parameters": [
        {
            "name": "first_step_fail", 
            "value": "True"
        }, 
        {
            "name": "second_step_fail", 
            "value": "False"
        }
    ], 
    "parentIds": [], 
    "labels": [
        {
            "name": "testClass", 
            "value": "test.steps.simple_steps_test"
        }
    ], 
    "stop": 1482258280772, 
    "statusDetails": {
        "message": "AssertionError: assert not True", 
        "trace": "first_step_fail = True, second_step_fail = False\n\n    @pytest.mark.parametrize(\"first_step_fail, second_step_fail\", true_false)\n    def test_sequence_of_outside_steps(first_step_fail, second_step_fail):\n>       first_step(first_step_fail)\n\ntest/steps/simple_steps_test.py:74: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nstep_fail = True\n\n    @pytest.allure.step(\"First step\")\n    def first_step(step_fail):\n>       assert not step_fail\nE       assert not True\n\ntest/steps/simple_steps_test.py:64: AssertionError"
    }, 
    "start": 1482258280772, 
    "steps": [
        {
            "name": "First step", 
            "description": null, 
            "descriptionHtml": null, 
            "steps": null, 
            "attachments": null, 
            "start": 1482258280772, 
            "stop": 1482258280772, 
            "id": null, 
            "status": "failed", 
            "statusDetails": null, 
            "parameters": null
        }
    ], 
    "fullName": "test.steps.simple_steps_test#test_sequence_of_outside_steps[True-False]", 
    "afters": [], 
    "id": "6e473fd350e444d5abbaca6407f12dfb", 
    "befores": []
}